src/test/scala/org/broadinstitute/dsde/agora/server/webservice/AgoraMethodsSpec.scala

```diff
@@ -107,11 +105,16 @@ class AgoraMethodsSpec extends ApiServiceSpec {
   }
 
   "Agora" should "store 10kb of github markdown as method documentation and return it without alteration" in {
-    Post(ApiUtil.Methods.withLeadingSlash, testAgoraEntityBigDoc) ~>
+    Post(ApiUtil.Methods.withLeadingSlash + "?includedField=documentation", testAgoraEntityBigDoc) ~>
       methodsService.postRoute ~> check {
       handleError(
         entity.as[AgoraEntity],
-        (entity: AgoraEntity) => assert(entity.documentation.get === bigDocumentation.get)
+        (entity: AgoraEntity) => {
+          val searchEntity = new AgoraEntity(name = entity.name, namespace = entity.namespace, snapshotId = entity.snapshotId)
```

Comment:
I think if you are hitting GET namespace/name/id you should get the whole entity, docs and payload included. If you are referencing a specific method, don't you want the entire method? Otherwise the only way for a caller to look at their documentation is to specify:
GET methods?namespace=mine&name=myMethod&snapshotId=X&includedField=documentation
vs
GET methods/mine/myMethod/snapshotId

I thought that was part of the motivation for keeping 2 GET endpoints- one for a list of truncated entities, configurable with entity projection. The other for getting the full details of a single method.

or does changing findSingle not affect that behavior?  I'm less concerned about the guts than about the user-facing endpoints